importer.py

```diff
@@ -1255,10 +1320,17 @@ def begin_child(self, context, type, name, values):
 
         #if type in ('SCALAR', 'CHAR*', 'BILLBOARD', 'DCS', 'DART', 'SWITCH', 'OBJECTTYPE', 'TAG', 'MODEL', 'TEXLIST', 'REF'):
 
+        if type in ('SWITCHCONDITION', 'SWITCH'):
+            return EggSwitchNode()
+
+        if type in ('DCS'):
+            # DCS tags won't have a name
+            self.properties[type] = values[0]
+
         if type in ('SCALAR', 'CHAR*'):
             name = name.lower().replace('_', '-')
 
-            if name in ('collide-mask', 'from-collide-mask', 'into-collide-mask', 'bin', 'draw-order'):
+            if name in ('collide-mask', 'from-collide-mask', 'into-collide-mask', 'bin', 'draw-order', 'scroll_u', 'scroll_v', 'alpha'):
```

Comment:
These scroll_u and scroll_v will never be detected since the line above replaces `_` with `-`

Reply:
Thanks for catching that, fixed!